lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -51,6 +64,96 @@ use tdp::{
     SharedDirectoryWriteResponse, TdpErrCode,
 };
 
+#[derive(Debug)]
+pub struct TeleportRdpdrBackend {
+    /// Active device ids for this session.
+    ///
+    /// The smartcard device id is always active, and always the first element in this vector.
+    active_device_ids: Vec<u32>,
```

Comment:
Do we need this to be `Vec`? AFAICT we only ever push single (constant) value here, can be `Option` probably or even `bool`

Reply:
Eventually we'll have more device ids -- when we announce a drive to redirect we add a device id, in principle we can redirect multiple at once, and may also add more device types in the future.

Granted this is not necessarily the best way to do it if/when we have more. For now I prefer to do it this admittedly somewhat stupid looking way because it will make the porting process more efficient (as this is what we have currently with rdp-rs). Once it's done with I'm happy to change it to a better system.

Reply:
Makes sense, thanks for clarifying